src/screens/auth/AuthScreen.tsx

```diff
@@ -1,14 +1,14 @@
 import React, {useEffect} from 'react';
-import {StyleSheet, View, Text} from 'react-native';
+import {View, Text} from 'react-native';
 import Input from '../../components/ui/Input';
-import {Colors} from '../../styles/Colors';
 import {Button} from '../../components/ui/Button';
 import {useForm} from 'react-hook-form';
 import {observer} from 'mobx-react';
 import {useRootStore} from '../../base/hooks/useRootStore';
 import {LoaderFlex} from '../../components/ui/Loader';
+import {auth_screen_styles} from '../styles';
 
-enum EAuthForm {
+enum EAUTHFORM {
```

Comment:
if enum name consist of 2 or more words, it's better to use camelCase for this cases